src/.clang-format

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
Did you mean to include this file? We already have a .clang-format in the project root. This one looks more like standard google formatting than souffle.

Reply:
Initially - yes. I dumped clang-format-4.0 default settings in there. 
The reason is that not all systems have clang-format-4.0. With the format file we are independent of 4.0 / any version can be used. However, I failed to make it work.